### PR TITLE
Change "What is Kubeflow" information from v0.2

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/website/issues/268

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/285)

### CODE REVIEW

File: content/docs/about/kubeflow.md [hunk 1]
```diff
@@ -29,11 +29,25 @@ Ultimately, we want to have a set of simple manifests that give you an easy to
 use ML stack _anywhere_ Kubernetes is already running, and that can self 
 configure based on the cluster it deploys into.
 
+## What is Kubeflow?
+
 Kubeflow is *the machine learning toolkit for Kubernetes*. 
 
-## What is Kubeflow?
+It's a collection of configuration files and scripts that you download,
```
Comment (abhi-g) [inline content/docs/about/kubeflow.md]: I prefer the existing description which provides a much higher level description of what Kubeflow is. That description allows underlying details of Kubeflow as a platform to evolve without requiring changes to what defines it.

Comment (richardsliu) [reply content/docs/about/kubeflow.md]: +1. Also Kubeflow has more components than the main repository (which is where the config files and scripts are). For example we also have TF training operators and serving.

Comment (andreyvelich) [reply content/docs/about/kubeflow.md]: @abhi-g @richardsliu thank you for your review. 
Should we add this information (from v0.2) bellow or just leave "What is Kubeflow" as it is?

Comment (abhi-g) [reply content/docs/about/kubeflow.md]: @andreyvelich I looked at the git issue from Sarah that you referenced. It seems like a more appropriate change would be to make a PR in the v0.2 branch of the docs to make this section identical to this paragraph in master.

Comment (abhi-g) [reply content/docs/about/kubeflow.md]: @andreyvelich I looked at the git issue from Sarah that you referenced. It seems like a more appropriate change would be to make a PR in the v0.2 branch of the docs to make this section identical to this paragraph in master.

Comment (andreyvelich) [reply content/docs/about/kubeflow.md]: @abhi-g Change it in v0.2 branch

**Comment by andreyvelich** [comment]
I signed it!

**Comment by andreyvelich** [comment]
@abhi-g I will create a new PR because here I got problem with CLA. 
It might be because I changed branch from master to v0.2-branch.

**Comment by abhi-g** [comment]
Makes sense. Plus making this change in v0.2 directly might be easier to
avoid a lot of conflicts arising due to other changes getting pulled in etc.

**Pull request closed without merging**